Ports/chibi-scheme/patches/0001-Exclude-tm_zone-for-serenity.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Paul Irwin <paulirwin@gmail.com>
+Date: Fri, 1 Sep 2023 17:00:45 -0600
+Subject: [PATCH] Exclude tm_zone for serenity
+
+---
+ lib/chibi/time.sld  | 2 +-
+ lib/chibi/time.stub | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/lib/chibi/time.sld b/lib/chibi/time.sld
+index 1fba628ecf3e0971c59b45f31d1892e4e485074c..8d9f575de68624f1065a419aed31148f555b0223 100644
+--- a/lib/chibi/time.sld
++++ b/lib/chibi/time.sld
+@@ -8,7 +8,7 @@
+           time-day-of-week time-day-of-year time-dst?
+           tm? timeval? timezone?)
+   (cond-expand
+-   (solaris)
++   ((or solaris serenity crossserenity))
```

Comment:
What is crossserenity?

Reply:
This was a remnant of various attempts at getting this to work. I thought I might have had to do a special cross-platform configuration like Chicken does when cross-compiling for Windows from Linux, but I got it working without doing that. I should be able to remove this, thanks!

Reply:
Removed.